config/core/test-configs.yaml

```diff
@@ -2327,6 +2327,21 @@ test_configs:
   - device_type: r8a774a1-hihope-rzg2m-ex
     test_plans:
       - baseline
+      - baseline-nfs
+      - kselftest-filesystems
+      - kselftest-futex
+      - kselftest-lib
+      - kselftest-lkdtm
+      - kselftest-seccomp
+      - ltp-crypto
+      - ltp-fcntl-locktests
+      - ltp-ima
```

Comment:
This one doesn't appear to be run on any platform except one x86 Chromebook, at least on staging.  It might need more investigation to see if that's by design or if something went wrong.

Reply:
works

Reply:
OK: https://staging.kernelci.org/test/plan/id/617a5db8ce6c18f259a6c45d/

A couple of test cases are failing, not sure if that's expected on this platform.